syfertext/tokenizer.py

```diff
@@ -138,11 +138,15 @@ def __call__(self, text: Union[String, str]):
         # I do not assign the Doc here any owner, this will
         # be done by the SupPipeline object that operates
         # this tokenizer.
-        doc = Doc(self.vocab)
+        doc = Doc(self.vocab, text)
 
         # The number of characters in the text
         text_size = len(text)
```

Comment:
no need for this line, you can use `len(text)` directly in the if statement. it is pretty clear.

Reply:
actually it was in use at 2 other places previously also!
anyway i updated them